src/agent/utilities.ts

```diff
@@ -15,6 +15,14 @@ export interface GetOrCreateResult<T> {
     created: boolean;
     result: T;
 }
+
+//converts inputString to titleCase string. For example, "abc def" is converted to "Abc Def"
+export function toTitleCase(inputString: string): string {
+    if (inputString) {
+        return inputString.replace(/\w\S*/g, function(str) { return str.charAt(0).toUpperCase() + str.substr(1).toLowerCase(); });
```

Comment:
If you are writing utility method, it should be independent. It needs it's own UT